data.sql

```diff
@@ -15,3 +15,28 @@ VALUES('Charmander', '2020-02-08', 0, true, -11),
     ('Ditto', '2022-05-14', 4, true, 22);
 //insert skipped animal data
 INSERT INTO ANIMALS(NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES('Squirtle','1993-04-02',3,false,-12.13);
+//insert data into owners table;
+INSERT INTO OWNERS(FULL_NAME,AGE)
+ VALUES('Sam Smith',34),
+ ('Jennifer Orwell',19),
+ ('Bob',45),
+ ('Melody Pond',77),
+ ('Dean Winchester',14),
+ ('Jodie Whittaker',38);
+ //insert data into species table;
+ INSERT INTO SPECIES (NAME) VALUES('Pokemon'),('Digimon');
+
+//Modify your inserted animals so it includes the species_id value:
+//If the name ends in "mon" it will be Digimon
+//All other animals are Pokemon
+UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
```

Comment:
- [ ] As I mentioned previously in my last review, you need to add both queries where you are updating the animals with `species_id = 1`, name ends with `%mon` and the rest of the animals where the `species_id` is null, with `species_id = 2`. I agree that things are working on your end perfectly fine, but it doesn't mean that it works on my side too. We as code reviewer, run yours queries step by step to get the same results that you got at your end with the help of screenshots that you added. That's why I told you, not only to add the queries but also update your `screenshots` in the PR description. Please read my previous comment carefully if you have any doubts. Your previous code reviewers also mentioned that you need to update your PR description with the latest screenshots, which in your case are still missing.

### Please justify the below screenshot and the query that I selected below:

#### In Screenshots
![image](https://github.com/ismayito/vet-clincic-database/assets/66476329/5f0cee49-79b4-4f57-96c0-2aa0d3c6b110)

#### In actual query
```sql
UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
```

**Kindly add the below queries to fulfil the requirements and update the screenshots in the PR description, which will help your next code reviewer to verify your work. It is very important to not miss any screenshots as per the milestone guidelines. After this you will left with 2 more review chances, so take your decision wisely and don't forget to implement the changes as I suggested.**


### Missing Queries (Look at the difference)
```suggestion
UPDATE ANIMALS SET species_id=1 WHERE TRIM(NAME) LIKE '%mon';
UPDATE ANIMALS SET species_id=2 WHERE species_id IS NULL;
```

### ⚠️ Your screenshots must match with your queries that you performed as per instructions, so add the updated screenshots in your PR. If you are still not convinced or not satisfied with the review, please use the second opinion form, which you can find in my review at the very end or [click here](https://airtable.com/apphXa7srZyRCpJVR/shrQAqnBwek5a0O0s)